README.md

```diff
@@ -241,6 +242,7 @@ See the [README](doc/README.md) file in the doc directory for more info.
   - [LimeChat](http://limechat.net/mac/)
   - [LittleSnitch](http://www.obdev.at/products/littlesnitch/)
   - [Livestreamer](http://livestreamer.tanuki.se/)
+  - [LSColors](https://github.com/trapd00r/LS_COLORS)
```

Comment:
Ok

On Friday, April 17, 2015, Laurent Raufaste <notifications@github.com>
wrote:

> In README.md <https://github.com/lra/mackup/pull/516#discussion_r28635278>
> :
>
> > @@ -241,6 +242,7 @@ See the [README](doc/README.md) file in the doc directory for more info.
> >    - [LimeChat](http://limechat.net/mac/)
> >    - [LittleSnitch](http://www.obdev.at/products/littlesnitch/)
> >    - [Livestreamer](http://livestreamer.tanuki.se/)
> > +  - [LSColors](https://github.com/trapd00r/LS_COLORS)
>
> just remove it, and use a custom config =)
>
> https://github.com/lra/mackup/tree/master/doc#add-support-for-an-application-or-any-file-or-directory
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/lra/mackup/pull/516/files#r28635278>.
>


-- 
Troy Wilson
Phone: 913.667.9662
Fax: 1.877.748.9636
Telegram Messenger: @Troy_Wilson
Troy@MerchantGuy.com

PGP Signed Emails and PGP Encryption available upon request.